build_tools/azure/jobs.yml

```diff
@@ -0,0 +1,70 @@
+jobs:
+- job: Windows
+
+  pool:
+    vmImage: 'vs2017-win2016'
+
+  strategy:
+    maxParallel: 6
+    matrix:
+      Python35-x86:
+        python.version: '3.5'
+        architecture: 'x86'
+
+      Python35-x64:
+        python.version: '3.5'
+        architecture: 'x64'
+
+      Python36-x86:
+        python.version: '3.6'
+        architecture: 'x86'
+
+      Python36-x64:
+        python.version: '3.6'
+        architecture: 'x64'
+
+      Python37-x86:
+        python.version: '3.7'
+        architecture: 'x86'
+
+      Python37-x64:
+        python.version: '3.7'
+        architecture: 'x64'
+
+  steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '$(python.version)'
+      architecture: '$(architecture)'
+    displayName: 'Setting Python version and system architecture'
+
+  - template: build_and_deploy.yml
+
+- job: macOS
+
+  pool:
+    vmImage: 'macOS-10.13'  # Also have the option of 10.14, if we want that
+
+  # Only need to set this for macOS
+  variables:
+    PMD_MPL_BACKEND: TkAgg
```

Comment:
Yep, forgot that's how we were [setting it in Travis](https://github.com/tgsmith61591/pmdarima/blob/master/pmdarima/utils/visualization.py#L23)

Reply:
The only thing I am not doing that Travis does is setting [`PMD_MPL_DEBUG`](https://github.com/tgsmith61591/pmdarima/blob/a133de78ba5bd68da9785b061f519ba28cd514cc/.travis.yml#L23). I don't really know how that is used, is that something we need?

Reply:
We shouldn't for now

Reply:
Also, it seems we set [`PMDARIMA_MPL_BACKEND`](https://github.com/tgsmith61591/pmdarima/blob/a133de78ba5bd68da9785b061f519ba28cd514cc/.travis.yml#L24) in Travis instead of `PMD_MPL_BACKEND` 🤔